src/components/export-modal/export-graph.js

```diff
@@ -21,9 +21,13 @@ const exportGraph = ({ format, theme, graphSize, mockFn }) => {
   clone.classList.add('kedro', `kui-theme--${theme}`, 'pipeline-graph--export');
 
   // Reset zoom/translate
-  let width = graphSize.width + graphSize.marginx * 2;
-  let height = graphSize.height + graphSize.marginy * 2;
-  clone.setAttribute('viewBox', `0 0 ${width} ${height}`);
+  let width, height;
+  const hasGraph = graphSize.width && graphSize.height;
```

Comment:
We might need to add the checks for valid values of `width` and `height` to eliminate those console errors?

Reply:
good point! fixed 👍